extensions/amp-subscriptions/0.1/test/test-entitlement-store.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Entitlement, Entitlements} from '../entitlements';
+
+import {EntitlementStore} from '../entitlement-store';
+
+describe('entitlement-store', {}, () => {
```

Comment:
This is defined somewhat incorrectly. The same problem is in the `test-amp-subscription-platform.js` and other test files - that's why none of the tests are actually executed :)

It should be either:
```
describe('name', () => {
  // body
});
```
or
```
describes.TYPE('name', {/*config*/}, () => {
  // body
});
```

`TYPE` here could be `sandboxed`, `realWin`, etc.

Reply:
Fixed